rustls/src/lib.rs

```diff
@@ -413,12 +418,6 @@
 //!
 //! - `zlib`: uses the `zlib-rs` crate for RFC8879 certificate compression support.
 //!
-//! ## Crate cfg options
-//!
-//! - `unstable_portable_atomic_arc` - configures this fork to use `Arc` from `portable_atomic_util` instead
-//!   of `std::sync::Arc` - requires Rust nightly together with `portable_atomic_unstable_coerce_unsized`
-//!   to build successfully.
```

Comment:
restore this section & use it to briefly document the unstable / unsupported cfg option